data/assets/examples/renderable/renderablemodel/model_animation.asset

```diff
@@ -0,0 +1,49 @@
+-- Animation
+-- This example loads a model with a simple keyframe animation. The animation starts at
```

Comment:
I'm not really sure how to explain the keyframe animation part in a way that does not make it more complex. I wanted to have the keyword "_keyframe_ animation" in there to make it somewhat clearer that we **only** support that type of animation but I could also leave it as only "animation" and leave the documentation for what types are supported.

Reply:
👍 I agree with you, in that it would make it more complex. Removing the "keyframe" would solve it, IMO. 

It's also future-proofing to remove it. Who knows, one day we might support other types of animations and then this example does not have to be updated ;)